test/functional/fixtures/api/es-next/take-screenshot/test.js

```diff
@@ -0,0 +1,40 @@
+var expect                  = require('chai').expect;
+var config                  = require('../../../../config.js');
+var checkScreenshotsCreated = require('../../../../assertion-helper.js').checkScreenshotsCreated;
+
+
+var testOnTravis = config.isTravisTask;
+
+
+describe('[API] t.takeScreenshot()', function () {
+    it('Should take a screenshot', function () {
+        return runTests('./testcafe-fixtures/take-screenshot.js', 'Take a screenshot',
+            { setScreenshotPath: true, only: 'local' })
+            .then(checkScreenshotsCreated)
+            .then(function (success) {
+                if (!testOnTravis)
```

Comment:
Yes, because of https://github.com/DevExpress/testcafe/pull/552/files#diff-15454f369c87c574b6885fdfb5bd6489R125